stories/tooltip.stories.js

```diff
@@ -0,0 +1,259 @@
+import React from 'react';
+import { Story, Demo, Description, Code, comment } from './storybook/blocks';
+import './tooltip.scss';
+
+const title = 'Tooltip';
+const component = 'Manifold__Tooltip';
+
+export default { title, component };
+
+export const defaultTooltip = () => (
+  <Story>
+    <Demo>
+      <div className="Manifold__Tooltip__Container" aria-describedby="tooltip-1">
+        <button type="button" className="Manifold__Button">
+          Hover me
+        </button>
+        <div role="tooltip" id="tooltip-1" className="Manifold__Tooltip">
+          Tooltip describing the trigger element.
+        </div>
+      </div>
+    </Demo>
+    <Description>
+      <h1>Default</h1>
+      <p>
+        Use tooltips to provide extra context to the user when hovering a trigger. The tooltip
+        contents default to one line of text. Default position is above the trigger.
+      </p>
+      <p>
+        Make sure to add the <code>aria-describedby="tooltip-id"</code> attribute to the element
+        triggering the tooltip. To use the tooltip without the trigger and position manually, omit
+        the `Manifold__Tooltip__Container` wrapping element.
```

Comment:
Sorry—we don’t have Markdown support here so the '`' needs to be `<code></code>` instead.

Reply:
![Screen Shot 2020-04-15 at 13 05 21](https://user-images.githubusercontent.com/1369770/79377989-c983f600-7f19-11ea-8da0-e845838fde21.png)